test/SILOptimizer/accessed_storage_analysis.sil

```diff
@@ -340,7 +340,7 @@ class C {
 
 // CHECK-LABEL: @readIdentifiedClass
 // CHECK: [read] Class %0 = argument of bb0 : $C
-// CHECK: Field: @_hasStorage var property: Int
```

Comment:
These are changed because this printer doesn't print for SIL. I could tell it to print for SIL, but that would cause other tests to print slightly differently, and this output isn't intended to convey full semantics.